bertopic/plotting/_barchart.py

```diff
@@ -93,6 +93,14 @@ def visualize_barchart(topic_model,
                    marker_color=next(colors)),
             row=row, col=column)
 
+        if len(words) > 12:
+            height = 250 + (len(words) - 12) * 11
+
+        if len(words) > 9:
+            fig.update_yaxes(
+                tickfont=dict(size=(height - 140) // len(words))
+            )
```

Comment:
At first I tried scaling the font size while keeping the figure height constant, which started making the text unreadable after a certain number of words, so I chose 12 as the tipping point for changing the figure size. 9 is the actual tipping point where the words start disappearing from the barchart. In case you would like me to support the previous behaviour I could add a parameter `autoscaling=True`?

Reply:
`autoscaling` seems like a good parameter. I want to prevent breaking changes as much as possible.